tests/Ims/DataTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace ATDev\RocketChat\Tests\Ims;
+
+use PHPUnit\Framework\TestCase;
+use AspectMock\Test as test;
+use ATDev\RocketChat\Ims\Data;
+
+class DataTest extends TestCase
```

Comment:
I don't see all the getter/setters tested separately here so we can check the validation. Is it needed?